docs/metadata.md

```diff
@@ -932,110 +932,166 @@ client.metadata.query(from, ancestorFolderId, options)
+								}
+							]
 						},
-						{
-							"type": "folder",
-							"id": "15286891196",
-							"sequence_id": "1",
-							"etag": "1",
-							"name": "North America"
+						"created_at": "2017-04-20T12:55:27-07:00",
+						"modified_at": "2017-04-20T12:55:27-07:00",
+						"trashed_at": null,
+						"purged_at": null,
+						"content_created_at": "2017-01-06T17:59:01-08:00",
+						"content_modified_at": "2017-01-06T17:59:01-08:00",
+						"created_by": {
+							"type": "user",
+							"id": "193973366",
+							"name": "Box Admin",
+							"login": "admin@company.com"
+						},
+						"modified_by": {
+							"type": "user",
+							"id": "193973366",
+							"name": "Box Admin",
+							"login": "admin@company.com"
+						},
+						"owned_by": {
+							"type": "user",
+							"id": "193973366",
+							"name": "Box Admin",
+							"login": "admin@company.com"
 						},
-						{
+						"shared_link": null,
+						"parent": {
 							"type": "folder",
 							"id": "16125613433",
 							"sequence_id": "0",
 							"etag": "0",
 							"name": "2017"
-						}
-						]
-					},
-					"created_at": "2017-04-20T12:55:27-07:00",
-					"modified_at": "2017-04-20T12:55:27-07:00",
-					"trashed_at": null,
-					"purged_at": null,
-					"content_created_at": "2017-01-06T17:59:01-08:00",
-					"content_modified_at": "2017-01-06T17:59:01-08:00",
-					"created_by": {
-						"type": "user",
-						"id": "193973366",
-						"name": "Box Admin",
-						"login": "admin@company.com"
-					},
-					"modified_by": {
-						"type": "user",
-						"id": "193973366",
-						"name": "Box Admin",
-						"login": "admin@company.com"
-					},
-					"owned_by": {
-						"type": "user",
-						"id": "193973366",
-						"name": "Box Admin",
-						"login": "admin@company.com"
-					},
-					"shared_link": null,
-					"parent": {
-						"type": "folder",
-						"id": "16125613433",
-						"sequence_id": "0",
-						"etag": "0",
-						"name": "2017"
+						},
+						"item_status": "active"
 					},
-					"item_status": "active"
-				},
-				"metadata": {
-					"enterprise_123456": {
-						"someTemplate": {
-						"$parent": "file_161753469109",
-						"$version": 0,
-						"customerName": "Phoenix Corp",
-						"$type": "someTemplate-3d5fcaca-f496-4bb6-9046-d25c37bc5594",
-						"$typeVersion": 0,
-						"$id": "ba52e2cc-371d-4659-8d53-50f1ac642e35",
-						"amount": 100,
-						"claimDate": "2016-04-10T00:00:00Z",
-						"region": "West",
-						"$typeScope": "enterprise_123456"
+					"metadata": {
+						"enterprise_12345": {
+							"someTemplate": {
+								"$parent": "file_161753469109",
+								"$version": 0,
+								"customerName": "Phoenix Corp",
+								"$type": "someTemplate-3d5fcaca-f496-4bb6-9046-d25c37bc5594",
+								"$typeVersion": 0,
+								"$id": "ba52e2cc-371d-4659-8d53-50f1ac642e35",
+								"amount": 100,
+								"claimDate": "2016-04-10T00:00:00Z",
+								"region": "West",
+								"$typeScope": "enterprise_123456"
+							}
```

Comment:
Hard to tell if anything changed other than the indent here.

Reply:
Just reformatting. No data changed